analysis/topEFT/run.py

```diff
@@ -83,7 +83,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
   # Read from cfg files
   for f in allInputFiles:
     if not os.path.isfile(f):
-      raise Exception(f'[ERROR] Input file {f} not found!')
+      raise Exception('[ERROR] Input file %s not found!'%f)
```

Comment:
Just curious why you removed the f-string?

Reply:
To be honest, I wasn't aware of the syntax and it was crashing (likely because I wasn't using the right python version when I made the change). But know that you mention it, indeed, this is a valid (and perhaps preferred) way of doing it, I will change it back to what was there before

Reply:
Thanks!